app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -104,10 +119,15 @@ export default {
           body
         );
         if (response.status === 201) {
-          this.showAlert(
-            'success',
-            'Your form has been successfully submitted (UX here TBD)'
-          );
+          this.$router.push({
+            name: 'FormSubmissionView',
+            params: {
+              formId: this.formId,
+              versionId: response.data.formVersionId,
+              submissionId: response.data.id,
+            },
+            query: { success: true },
```

Comment:
Likely not going this way in the future (don't think we can navigate to this page when a non-creating user fills out the form) so don't worry about this query param

Reply:
This is going away next pr